README.md

```diff
@@ -1,6 +1,100 @@
-# odin-recipes
-Microverse Pre-Enrollment Bootcamp assignment on HTML-CSS :=> Odin Recipes
+<a name="readme-top"></a>
 
-```text
-Coded By: Md Mahabub A. (~/gh/mahabubx7)
-```
+
+<div align="center">
+
+  <br/>
+
+  <h3><b>Odin Recipes - Code review practice project </b></h3>
+
+</div>
```

Comment:
I would like to **suggest** making these markups good by removing unnecessary spaces.